tests/integration/internal/conftest.py

```diff
@@ -0,0 +1,24 @@
+import pytest
+from spccore.internal.cache import *
+
+
+@pytest.fixture
+def cache() -> Cache:
+    cache = Cache()
```

Comment:
Create a temporary test cache dir so that when testing locally, the default cache won't be cleared by purge

Reply:
good idea. will do.